scripts/install-helpers/baremetal-coco/install.sh

```diff
@@ -0,0 +1,733 @@
+}
+
+function create_amd_node_feature_rules() {
+    echo "Node Feature Discovery operator | creating amd node feature rules"
+
+    pushd nfd || return 1
+    oc apply -f amd-rules.yaml || return 1
+    popd || return 1
+
+    echo "Node Feature Discovery operator | node feature rules successfully created"
+}
+
+# Function to create KataConfig based on TEE type
+function create_kataconfig() {
+    local tee_type=${1}
+    local label='coco: "true"'
+
+    case $tee_type in
+    tdx)
+        label='intel.feature.node.kubernetes.io/tdx: "true"'
+        ;;
+    snp)
+        label='amd.feature.node.kubernetes.io/snp: "true"'
+        ;;
+    esac
+
+    # Create KataConfig object
+    oc apply -f - <<EOF
+apiVersion: kataconfiguration.openshift.io/v1
+kind: KataConfig
+metadata:
+  name: cluster-kataconfig
+spec:
+  enablePeerPods: false
+  logLevel: info
+  kataConfigPoolSelector:
+    matchLabels:
+      $label
+EOF
+
+    result=$?
+    [ $result -eq 0 ] || return 1
+
+    echo "KataConfig object successfully created"
+    return 0
+}
+
+# Function to set aa_kbc_params for teh Kata agent to be used for
+# attestation
+# Use drop-in configuration via MachineConfig
+# This function accepts the TEE type as an argument
+# It also accepts the Trustee URL as an argument
+# Trustee URL should be complete URL with the form http(s)://<trustee_ip>:<port>
+function set_aa_kbc_params_for_kata_agent() {
+    local tee_type=${1}
+    local trustee_url=${2}
+    local source=""
+    local filepath=""
+
+    # Create kata configuration toml override for the kernel_params
+    # Input kernel_params="agent.aa_kbc_params=cc_kbc::$trustee_url"
+    kata_override="[hypervisor.qemu]
+kernel_params= \"agent.aa_kbc_params=cc_kbc::$trustee_url\""
+
+    # Create base64 encoding of the drop-in to be used as source
+    source=$(echo "$kata_override" | base64 -w0) || return 1
+
+    # This is applied after Kata installation so we should use the kata-oc label
+    # for worker nodes
+    local mc_label="machineconfiguration.openshift.io/role: kata-oc"
+
+    if is_single_node_ocp; then
+        mc_label="machineconfiguration.openshift.io/role: master"
+    fi
+
+    case $tee_type in
+    tdx)
+        filepath=/etc/kata-containers/tdx/config.d/96-kata-kernel-config
+        ;;
+    snp)
+        filepath=/etc/kata-containers/snp/config.d/96-kata-kernel-config
+        ;;
+    esac
+
+    cat <<EOF >./96-kata-kernel-config-mc.yaml
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    $mc_label
+  name: 96-kata-kernel-config
+  namespace: openshift-machine-config-operator
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,$source
+        mode: 420
+        overwrite: true
+        path: $filepath
+EOF
+
+    oc apply -f 96-kata-kernel-config-mc.yaml || return 1
+
+}
```

Comment:
The `|| return 1` isn't needed.

```suggestion
    oc apply -f 96-kata-kernel-config-mc.yaml
}
```